assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -170,15 +176,16 @@ jobs:
     resources:
       instance_count: ${{parent.inputs.instance_count}}
     max_concurrency_per_instance:  ${{parent.inputs.max_concurrency_per_instance}}
-    mini_batch_size: "100KB"
+    mini_batch_size: "10GB"
```

Comment:
Do we want to change this for all model types ?

Reply:
I tried to make this configurable by input parameter, however no matter how I try it I get the following error `Met error <class 'ValueError'>:Parameter mini_batch_size must follow regex rule ^\d+([kKmMgG][bB])*$`. It seems that this is not configurable via pipeline inputs so the user will need to register new components if they wish to change the mini_batch_size 10GB is very large and should cover most use cases.

Reply:
@yuanzhuangyuanzhuang , is it fine to update mini_batch_size here to "10GB"? For NLP tasks, it would mean there is only 1 mini batch as datasets are rarely this large.

Reply:
@arun-rajora @yuanzhuangyuanzhuang Is there any way to configure the mini_batch_size with input params?

Reply:
No. We cannot made it to 10GB as the AOAI quality job will get timeout. It cannot be set as parameters as it will check the input whether match some regex. Let me figure out how to put this as a parameter Or release a new perf component set this to 10GB.

Reply:
reverted to 100KB for this PR. For vision model benchmarking internally we can register new components to run the pipeline